inc/CrvInfoHelper.hh

```diff
@@ -60,7 +66,25 @@ namespace mu2e
         const art::Ptr<SimParticle> &parentParticle = simParticle->hasParent() ? simParticle->parent() : simParticle;
         return parentParticle->hasParent() ? parentParticle->parent() : parentParticle;
       }
-
+      static const std::map<int,int> GetSiPMMap(GeomHandle<CosmicRayShield> &CRS) // Helper function for FillCrvDigiInfoCollections and FillCrvPulseInfoCollections
+      {
+        // Create SiPM map to extract sequantial SiPM IDs
+        const std::vector<std::shared_ptr<CRSScintillatorBar> > &counters = CRS->getAllCRSScintillatorBars();
+        std::vector<std::shared_ptr<CRSScintillatorBar> >::const_iterator iter;
+        int iSiPM = 0;
+        std::map<int,int> sipm_map;
+        for(iter=counters.begin(); iter!=counters.end(); iter++)
+        {
+          const CRSScintillatorBarIndex &barIndex = (*iter)->index();
+          for(int SiPM=0; SiPM<4; SiPM++)
```

Comment:
You should also use a proper identifier for this literal 4 and for the literal 4 a few lines down.  This happens to be a different 4 than one you addressed earlier today.  I believe that this 4 is CRVId::nChanPerBar, (which is only accidentally the same value as CRVId::nLayers).  This is one of the reasons we want people to use named constexpr or variables, not numerical literals.

@ehrlich-uva please let us know whether or not my 4-ology is correct.

@ehrlich the construction ( something.barIndex.asInt()*4 +SIPM) appears several times in this code.  And I bet it appears in other places.  Do you provide a helper function that wraps this calculation?  If so, where is it?

@sam-grant if Ralf has such a function, please use it.  If not leave it for another day.

Reply:
Hi Rob. I agree that this is a different 4 to the number of layers, and I agree that `CRVId::nChanPerBar` looks like the right number to use, since there are 4 SiPMs per scintillator bar. @ehrlich-uva can you confirm? Thanks.

Reply:
I see that this calculation also appears in `CrvInfoHelper.cc` as you said, I'll change that to use the identifier as well.

Reply:
I just pushed the change from `4` to `CRVId::nChanPerBar`.

Reply:
Thanks.  I will approve it now.